app/services/calendar_factory_from_event_hash.rb

```diff
@@ -24,20 +26,19 @@ def create
 
 private
 
+  def parent_calendar
+    Calendar.find_by(calendar_area: @area, template: true) || Calendar.find_by(calendar_area: @area.parent_area, template: true)
```

Comment:
THis only allows for 3 levels of calendars

Reply:
I think this class should be clearer about what it does. We only use it in the code (rather than the specs) to create template area calendars so I think that's all it should do. That way we just need to pass in one area and get its template calendar.

Reply:
I agree - I think this should be done when we tackle (re-tackle) calendar areas